airbyte-integrations/connectors/source-google-search-console/source_google_search_console/spec.json

```diff
@@ -112,10 +113,40 @@
         ]
       },
       "custom_reports": {
-        "order": 4,
-        "type": "string",
         "title": "Custom Reports",
-        "description": "A JSON array describing the custom reports you want to sync from Google Search Console. See our <a href='https://docs.airbyte.com/integrations/sources/google-search-console'>documentation</a> for more information on formulating custom reports."
+        "description": "You can add your Custom Analytics report by creating one.",
```

Comment:
can we retain the description of the previous field?

Reply:
The description field is retained automatically if we switch to the previous version.